packages/cli/src/lib/textile/user-auth.ts

```diff
@@ -3,6 +3,7 @@ import { print } from "gluegun";
 
 const requireKeytar = () => {
   try {
+    // eslint-disable-next-line @typescript-eslint/no-require-imports
     return require("keytar");
```

Comment:
Yah we should probably just get rid of this textile integration since it isn't complete. We can come back to the history and finish the integration.

Reply:
Feel free to just leave this alone :P I can fix this

Reply:
Yeah, wasn't planning on touching that. I mean, if you wan't to load some expensive dependency on demand, it's perfectly fine to use require. That is also valid for optional/peer dependencies. But those are usually rare occurencies so we can just disable rule on those particular exceptions.